doc/StaticWorkflow.md

```diff
@@ -0,0 +1,275 @@
+# Preferences Edge Proxy Refinement I
+
+This describes the requests, responses, and payloads for the workflow where a
+static site makes save/retrieve requests of a secured preferences server.  For
+example, the static site may have incorporated UIO+ into their pages.  The user
+wants to save or retrieve their UIO+ preferences.
+
+The outline of the workflow is taken verbatim from the Fluid wiki page titled
```

Comment:
I think it would make sense to recontectualize the workflow here, as portions that talk about GPII make it a bit confusing. For example in point 2. it says "implemented by the GPII authorization server" which is something that will no longer be happening, and seems like it will be done via Google or other SSO provider.